### PR TITLE
Add topicGettingStarted sample

### PR DESCRIPTION
**Pull request by ShivangiReja**
Add topicGettingStarted sample #68

### CODE REVIEW

File: examples/samples/topicsGettingStarted.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  const client = ns.createTopicClient(topic);
+
+  const subscription1Client = ns.createSubscriptionClient(topic, subscription1);
+  const subscription2Client = ns.createSubscriptionClient(topic, subscription2);
+  const subscription3Client = ns.createSubscriptionClient(topic, subscription3);
+
+  //retrieving message one by one from each subscription
+  receiveMessage(subscription1Client, "color:red;");
+  receiveMessage(subscription2Client, "color:green;");
+  receiveMessage(subscription3Client, "color:cyan;");
```
Comment (ramya-rao-a) [inline examples/samples/topicsGettingStarted.ts]: What is the purpose of these colors?

Comment (ShivangiReja) [reply examples/samples/topicsGettingStarted.ts]: We are retrieving messages one by one from  each subscription and passing color variable to differentiate the subscription data(just like .Net sample). See the output and Please let me know if I need to change anything.
topicgettingstarted

Comment (ramya-rao-a) [reply examples/samples/topicsGettingStarted.ts]: `client.name`

File: examples/samples/topicsGettingStarted.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const topic = process.env.TOPIC_NAME || "";
+const subscription1 = process.env.SUBSCRIPTION_NAME1 || "";
+const subscription2 = process.env.SUBSCRIPTION_NAME2 || "";
+const subscription3 = process.env.SUBSCRIPTION_NAME3 || "";
+
+console.log("str: ", str);
+console.log("path: ", topic);
```
Comment (AlexGhiondea) [inline examples/samples/topicsGettingStarted.ts]: I think we should log the 3 subscriptions as well.

Comment (ShivangiReja) [reply examples/samples/topicsGettingStarted.ts]: Ok!

**Pull request merged**